ifu_session/Cube_Analysis_Webbinar_solutions.ipynb

```diff
@@ -8,14 +8,14 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    
</pre><p>This cell has been giving the following warnings:</p><pre class="ql-syntax" spellcheck="false">/Users/rgeda/miniconda3/lib/python3.7/site-packages/glue/core/data.py:1803: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use <code>arr[tuple(seq)]</code> instead of <code>arr[seq]</code>. In the future this will be interpreted as an array index, <code>arr[np.array(seq)]</code>, which will result either in an error or a different result.
  full_result[result_slices] = result

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>